.gitignore

```diff
@@ -1,5 +1,15 @@
 node_modules/
+.nyc_output/
+coverage/
+
 npm-debug.log
 .DS_Store
+
+#IDES
 .vscode/
+.idea/
+
 .reify-cache/
+
+#testing
+coverage.lcov
```

Comment:
I'm assuming these .gitignore files should largely be the same, it looks like `conflator`'s ignore file and this one doesn't match:
https://github.com/dailymuse/conflator/blob/develop/.gitignore

@qjflores any thoughts on which one of these files we should make the standard?

Reply:
conflator